types/encoding.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/0xPolygon/polygon-edge/helper/hex"
 )
 
+// ParseUint64orHex parses the given string as uint64 in hex
+// It should go to the common package from the logical perspective
+// as well as avoiding cycle imports.
+// DEPRECATED. Use common.ParseUint64orHex.
 func ParseUint64orHex(val *string) (uint64, error) {
```

Comment:
Should we remove it as part of EIP-1559 effort?

Reply:
I would do that separately once this PR is merged